koji_wrapper/base.py

```diff
@@ -33,6 +33,7 @@ def topurl(self):
     @topurl.setter
     def topurl(self, topurl):
         self.__topurl = topurl
+        self.__pathinfo = koji.PathInfo(topurl)
```

Comment:
I guess the lazy load model might be to check for is not None before assigning or setting to None on topurl change

Reply:
After irc discussion, I don't think we want this - we should initialize only if/when we need to build a url, as that is the only thing that uses the PathInfo object

Reply:
done collapsed to private lazy load member variable in KojiWrapper.  Initialized when first used.

This covers expected use cases but does not fully cover if you change a topurl after calling srpm_url() which is ok I guess as that's not something you would normally do.